### PR TITLE
fix: move spaces between tag and text out of tags (fix #548)

### LINKED ISSUE #548: Bold syntax is broken when select text with a space in wysiwyg
## Version
v1.4.3

## Test Environment
Chrome

## Current Behavior

1. In Wysiwyg, select the test including space at the end.
image

2. Apply bold using toolbar.
image

3. Change mode to Markdown.
image

Bold syntax is broken in Markdown.

## Expected Behavior

Bold Syntax should not be broken.

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

* Issue related to #548 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/toMarkRenderer.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import toMark from 'to-mark';
 import domUtils from './domUtils';
 
+const FINE_BEFORE_AND_AFTER_SPACES_RX = /^(\s*)(\S|\S.*\S)(\s*)$/;
```
Comment (dongwoo-kim) [inline src/js/toMarkRenderer.js]: 이건.. 그냥 함수 안으로 넣는 게 어떨까요? 딱히 공용으로 쓰일 변수가 아닌데..

Comment (seonim-ryu) [reply src/js/toMarkRenderer.js]: 처음에 정규식이 길어서 밖으로 뺐는데 상수명이랑 비슷비슷해서.. 넣어도 되겠네요ㅎ

File: test/unit/convertor.spec.js [hunk 2]
```diff
@@ -447,6 +447,70 @@ describe('Convertor', () => {
...
+        expect(toMark('foo&nbsp; &nbsp;<b>bar</b>baz')).toBe('foo\u00a0 \u00a0**bar**baz');
+        expect(toMark('foo&nbsp; &nbsp;<strong>bar</strong>baz')).toBe('foo\u00a0 \u00a0**bar**baz');
+
+        // italic
+        expect(toMark('foo&nbsp; &nbsp;<i>bar</i>baz')).toBe('foo\u00a0 \u00a0*bar*baz');
+        expect(toMark('foo&nbsp; &nbsp;<em>bar</em>baz')).toBe('foo\u00a0 \u00a0*bar*baz');
+
+        // strike
+        expect(toMark('foo&nbsp; &nbsp;<s>bar</s>baz')).toBe('foo\u00a0 \u00a0~~bar~~baz');
+        expect(toMark('foo&nbsp; &nbsp;<del>bar</del>baz')).toBe('foo\u00a0 \u00a0~~bar~~baz');
```
Comment (dongwoo-kim) [inline test/unit/convertor.spec.js]: 공백이 태그 안쪽에 들어가 있어야 할 것 같은데, 밖으로 나와 있네요.

Comment (seonim-ryu) [reply test/unit/convertor.spec.js]: TC 옯기다가 밖으로 빼버렸네요 =.= 매의 눈ㅎ

**Pull request merged**